cpp/src/arrow/dataset/CMakeLists.txt

```diff
@@ -104,6 +105,7 @@ if(NOT WIN32)
   add_arrow_dataset_test(filter_test)
   add_arrow_dataset_test(partition_test)
   add_arrow_dataset_test(scanner_test)
+  add_arrow_dataset_test(file_ipc_test)
```

Comment:
Nit, but can we follow lexicographic order?

Reply:
will do